plasmapy/particles/__init__.py

```diff
@@ -46,3 +57,30 @@
 deuteron = Particle("D 1+")
 triton = Particle("T 1+")
 alpha = Particle("He-4 2+")
+
+
+def json_load_particle(json_file_object):
+    """
+    Returns the appropriate object from input JSON file object
+
+    Parameters
+    ----------
+    json_file_object: file object
+        file object of the JSON file that contains the JSON string describing
+        the particle
+    """
+    json_string = None
+    json_string = json_file_object.read()
+    return json_loads_particle(json_string)
+
+
+def json_loads_particle(json_string):
```

Comment:
Use the same [signature as `json.loads`](https://docs.python.org/3/library/json.html#json.loads).